supplementary/devices/yuga.rst

```diff
@@ -25,45 +25,53 @@ The following entries are just a placeholder, exactly as this sentence.
      - Kernel
      - What works
      - What doesn't work
-   * - LineageOS placeholder `device page placeholder <placeholder>`_
-     - `android_device_placeholder <placeholder>`_
-     - `android_kernel_placeholder <placeholder>`_ based on vX.Y.Z
-     - ?
-     - ?
-   * - Ubports placeholder `device page placeholder <placeholder>`_
-     - `android_device_placeholder <placeholder>`_
-     - `android_kernel_placeholder <placeholder>`_ based on vX.Y.Z
-     - ?
-     - ?
+   * - `Yuga on LineageOS Wiki <https://wiki.lineageos.org/devices/yuga/>`_
+     - `LineageOS/android_device_sony_yuga <https://github.com/lineageos/android_device_sony_yuga>`_
+     - `LineageOS/android_kernel_sony_apq8064 <https://github.com/lineageos/android_kernel_sony_apq8064>`_ based on v3.4.113
+     - everything
+     -
 
 
 Kernel & Hardware
 ^^^^^^^^^^^^^^^^^
 
-Mainline (vX.Y.Z as of writing)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Mainline (v4.15-rc9 as of writing)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The kernel will start and there is already UART and USB debugging implemented
+for this device. There also has been worked on riva, which is AFAIK for
```

Comment:
Maybe add a link to 'riva' instead of using "As Far as I Know". Only say what you know is true.